<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>140</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>UART COMMUNICATION</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>UART COMMUNICATION</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>351</width>
      <height>104</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="UARTLAYOUT">
     <item row="1" column="2">
      <widget class="QPushButton" name="btn_closeUart">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_25">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>BaudRate</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="baudRateBox"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_26">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>COM</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="ComUart"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btn_SetUart">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_send">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>RUN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>391</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 5px;
border-style:double;
border-color: black;
background-color:rgb(227,227,227);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_33">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>301</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 5px;
border-style:double;
border-color: black;
background-color:rgb(227,227,227);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_34">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>300</y>
      <width>112</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Weight:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>290</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>538</x>
      <y>220</y>
      <width>141</width>
      <height>159</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="dataBitsBox"/>
     </item>
     <item>
      <widget class="QComboBox" name="parityBox"/>
     </item>
     <item>
      <widget class="QComboBox" name="flowControlBox"/>
     </item>
     <item>
      <widget class="QComboBox" name="stopBitsBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>220</y>
      <width>141</width>
      <height>159</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_27">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>DataBits</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_28">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>FlowControl</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_30">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Parity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_31">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>StopBits'</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>label_23</zorder>
   <zorder>label_24</zorder>
   <zorder>label_29</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>flowControlBox</zorder>
   <zorder>stopBitsBox</zorder>
   <zorder>parityBox</zorder>
   <zorder>dataBitsBox</zorder>
   <zorder>label_27</zorder>
   <zorder>label_28</zorder>
   <zorder>label_30</zorder>
   <zorder>label_31</zorder>
   <zorder>btn_send</zorder>
   <zorder>label_33</zorder>
   <zorder>label_34</zorder>
   <zorder>lcdNumber</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
